### ISSUE TITLE
Todo's for Poster & Paper

### ISSUE THREAD

**Issue by JMitnik** [JMitnik/NLP-Posterior-Collapse-and-Probing#4]
# Todo-do's in general

## Codewise

- [x] Fix testing code [KL?]
- [x] Do results (and readd multi-samples)
- [x] Get better qualitative samples
- [x] Prep code / Clean code
- [x] Move code to Notebook

## Writing

- [ ] Doublecheck introduction (0.6 page) (created draft)

- [ ] Related work (0.4 page): **(created draft)**
  - [ ] Add Bowman, **(mentioned)**
  - [ ] Kingsma's 2016 paper (starts with "Improved" not "Improving") **(mentioned)**
  - [ ] Mu-Forcing paper **(mentioned)**
  - [ ] The original VAE paper **(mentioned)** 

- [ ] Approach (1 page) **(created draft)**
  - [ ] Description of the task: Language Modeling **(created draft)**
  - [ ] Description of the RNNLM and Sentence VAE architectures **(Should be in experimental setup)**
  - [ ] Description of the three posterior collapse methods used in this paper **(created drafts)**

- [ ] Experiments and Results (1 page)
  - [ ] Data used, fold split
  - [ ] Metrics collected
  - [ ] Hyperparameters: which ones we varied, and a brief description of what we kept fixed
  - [ ] Results

- [ ] Discussion (1 page)
  - [ ] Word dropout and its failure; need to understand whether this is because of the amount of data, or our implementation
  - [ ] KL increased, which is great, seems to be consistent though
  - [ ] Best performing model by perplexity also has highest KL; this is good.
  - [ ] Interesting that mu-forcing does not lead to better NLL-Loss.

- [ ] Appendix
  - [ ] Interpolations

### Done

- [x] Poster
- [x] Decide on generic poster layout
- [x] Decide what to put in poster
- [x] Put general introduction in poster

- [x] RNN Loss [Pieter]
- [x] Mu-forcing extract mu [Pieter]
- [x] Hyperparameter selection for different cross validation [Jonathan]
	- **Fixed**: all model params are fixed.
	- **Variant**: hyper-params are the dropout ones and freebies and mu_force_beta.
  - [x] Adjust summarywriter to write for each parameter to different run-folder

- [x] Freebits double-check [Jonathan]

- [x] Multi-sample estimates of log-likelihood for VAE [Jonathan]

- [x] Ablation study [Pieter] <- Wait for summarywriter by Jonathan
